src/compiler/checker.ts

```diff
@@ -15396,20 +15368,6 @@ namespace ts {
             return symbol && getExportSymbolOfValueSymbolIfExported(symbol).valueDeclaration;
         }
 
-        function instantiateSingleCallFunctionType(functionType: Type, typeArguments: Type[]): Type {

```

Comment:
Thanks for pointing that out @sandersn, I sent out a linting PR if you'd like to take a look: https://github.com/Microsoft/TypeScript/pull/6140